binding/react-native/src/porcupine.tsx

```diff
@@ -32,11 +34,13 @@ class Porcupine {
    * @returns An instance of the engine.
    */
   public static async fromKeywords(
+    accessKey: string,
```

Comment:
I think to bring this in line with our other SDKs, we should use an enum for built-in keywords and call this function `fromBuiltInKeywords`. Also, "default keywords" -> "built-in keywords" and when you're providing a path to a file we're going to call it a "custom keyword"